perceval/backends/jira.py

```diff
@@ -163,6 +167,34 @@ def parse_issues(raw_page):
         for issue in issues:
             yield issue
 
+    @staticmethod
```

Comment:
I don't see these two methods as methods of this class. They are really specific to how fields are parsed. As we don't have a parser here I think it should be better to put them outside of this class, keeping them as module functions.
What do you think?